bench/journeys_intended_untracked.go

```
@@ -30,50 +31,171 @@ func mixedIntendedUntrackedCandidate(sandbox *Sandbox) error {
+	}
+	arguments := append([]string{"review", descriptor.OperationToken}, descriptor.ArgumentTokens...)
+	values := map[string][]string{"cwd": {r.sandbox.Repo}, "untracked_scope": {scope}, "intended_untracked": paths}
+	for _, slot := range descriptor.Values {
+		index := slot.SubstitutionLocation + 2
+		placeholder := "{{" + slot.Slot + "}}"
+		if slot.Repeated {
+			replacements := make([]string, len(values[slot.Slot]))
+			for valueIndex, value := range values[slot.Slot] {
+				replacements[valueIndex] = strings.Replace(arguments[index], placeholder, value, 1)
+			}
+			arguments = append(arguments[:index], append(replacements, arguments[index+1:]...)...)
```

golangci-lint (2.12.2)

[error] append to slice `replacements` with non-zero initialized length

(makezero)